mrbeam_ledstrips/state_animations.py

```diff
@@ -1131,11 +1133,14 @@ def _get_frame_duration(self, fps):
 
 
 	def _get_int_val(self, value):
+		# Implements a last_valid_progress value to avoid wrong progress values
+		# Will be handled properly in SW-3799. This is just a quick fix.
 		try:
 			value = int(float(value))
+			self.last_valid_progress=value
 		except:
-			self.logger.exception("_get_int_val() Cant convert value '%s' to int. Using 0 as value. ", value)
-			return 0
+			self.logger.debug("_get_int_val() Cant convert value '%s' to int. Using 0 as value. ", value)
```

Comment:
```suggestion
                         # TODO: This is just a quick fix. It will be handled in SW-3799.
			self.logger.debug("_get_int_val() Cant convert value '%s' to int. Using 0 as value. ", value)
```